content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,93 @@
+
+1. Updates to the API can be generated in more agile manner where documentation are embedded as description to any endpoints, parameters, and many more such as contact information, license, terms of use.
+2. Consumer of this API can also consume this API in an agile manner using the converter from this openAPI format to any language that were used as part of the their automation or CI/CD workflow. (Please check <https://openapi.tools> for more information for the API tools to generate client code)
+
+## REST API Details
+
+Any customer can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following link:
+
+[Link to the API repository](https://docs.ccs.arubathena.com/)
+
+
+
+### Documentation and Tutorials
+
+The overview of the Data Services Cloud Console is at [this link](https://www.hpe.com/us/en/storage/data-services-cloud-console.html#overview). There will be more blogs that will provide more information to help customer adopting this API.
+
+### Supported API Categories (Services)
+
+1. DataOps Manager
+2. Event Audit
+3. Task tracking
+4. Search
+5. Helps, Issues and announcement
+
+### Versioning
+
+The major version number will be provided in the resource path as v1 in this example:
+
+```
+/api/v1/<resource group>/...
+```
+
+Clients will be able to adopt the backward compatibility from the higher major version incremental.  However, both the new and old version of API will be supported until the announcement of the deprecation. Nonetheless, the older major version will always be frozen with exception of bug fixes. There will also be announcement of deprecation in the header and sunset header. 
+
+```
+/api/v1/<resource group>/...
+/api/v2/<resource group>/...
+```
+
+### HTTP Request Methods
+
+| HTTP Verbs | Description                                                         |
+| ---------- | ------------------------------------------------------------------- |
+| GET        | Retrieves target resource                                           |
+| POST       | Creates an entity or changes state                                  |
+| PUT        | Replaces target resource with data part of the HTTP Request payload |
+| DELETE     | Remove the target source                                            |
+
+### Relationship Authorization
+
+The client can only receive the properties that are authorized based on the Role Base Access Control for the user who created the access token. 
+
+### Asynchronous Response
+
+All of the REST API operations are stateless in nature, such as POST, in that scenario the task resource will return a response with HTTP code 202 "Accepted" and the reference to the task as follows:
+
+```
+Response: 202 Accepted
+{
+   "taskURi": "/api/v1/tasks/123e4567-e89b-12d3-a456-4266141741230"
+}
+```
+
+### Any Questions on Data Services Cloud Console API?
+
+Please join [HPEDEV Slack Workspace](https://slack.hpedev.io/) and start a discussion in our #DSCC channel
```

Comment:
URL to channel: https://hpedev.slack.com/archives/C02D6H623JP
I created it as Data Services Cloud Console, not API, but I mentioned API in the description. Feel free to suggest a better description. Make sure you keep an eye on it going forward and invite appropriate folks.

Reply:
Yes Didier, I agree with your suggestion, I think DSCC will be the perfect title for this platform

Reply:
Don't forget the URL to the channel, and I think it would be better to spell out the name "in our #Data-Services-Cloud-Console channel".

Reply:
I'd recommend changing title to "Any Questions on Data Services Cloud Console" since we remove the API from the name of the channel, it's more consistent.